src/Games/NexusMods.Games.AdvancedInstaller.UI/Content/Right/SelectLocation/TreeEntry/SelectableTreeEntryView.axaml.cs

```diff
@@ -1,39 +1,50 @@
 ﻿using System.Diagnostics.CodeAnalysis;
 using System.Reactive.Disposables;
+using System.Reactive.Linq;
 using Avalonia.ReactiveUI;
 using ReactiveUI;
 
-namespace NexusMods.Games.AdvancedInstaller.UI.Content.Right.Results.SelectLocation.SelectableDirectoryEntry;
+namespace NexusMods.Games.AdvancedInstaller.UI.SelectLocation;
 
 [ExcludeFromCodeCoverage]
-public partial class TreeEntryView : ReactiveUserControl<ITreeEntryViewModel>
+public partial class SelectableTreeEntryView : ReactiveUserControl<ISelectableTreeEntryViewModel>
 {
-    public TreeEntryView()
+    public SelectableTreeEntryView()
     {
         InitializeComponent();
 
         this.WhenActivated(disposable =>
         {
-            if (ViewModel == null)
-                return;
-
-            InitView();
+            // When the ViewModel not null, initialize the view.
+            this.WhenAnyValue(view => view.ViewModel)
+                .WhereNotNull()
+                .Do(InitView)
+                .Subscribe()
+                .DisposeWith(disposable);
 
-            this.BindCommand(ViewModel, vm => vm.LinkCommand, v => v.SelectRoundedButton)
+            this.BindCommand(ViewModel, vm =>
+                    vm.CreateMappingCommand, v => v.SelectRoundedButton)
                 .DisposeWith(disposable);
 
-            this.BindCommand(ViewModel, vm => vm.EditCreateFolderCommand, v => v.CreateFolderButton)
+            this.BindCommand(ViewModel, vm =>
+                    vm.EditCreateFolderCommand, v => v.CreateFolderButton)
                 .DisposeWith(disposable);
 
-            this.BindCommand(ViewModel, vm => vm.SaveCreatedFolderCommand, v => v.SaveCreatedFolderButton)
+            this.BindCommand(ViewModel, vm =>
+                    vm.SaveCreatedFolderCommand, v => v.SaveCreatedFolderButton)
                 .DisposeWith(disposable);
 
-            this.BindCommand(ViewModel, vm => vm.CancelCreateFolderCommand, v => v.CancelCreateFolderButton)
+            this.BindCommand(ViewModel, vm =>
+                    vm.CancelCreateFolderCommand, v => v.CancelCreateFolderButton)
                 .DisposeWith(disposable);
 
-            this.BindCommand(ViewModel, vm => vm.DeleteCreatedFolderCommand, v => v.DeleteCreatedFolderButton);
+            this.BindCommand(ViewModel, vm =>
+                    vm.DeleteCreatedFolderCommand, v => v.DeleteCreatedFolderButton)
+                .DisposeWith(disposable);
 
-            this.Bind(ViewModel, vm => vm.InputText, v => v.CreateFolderNameTextBox.Text)
+            // Two way binding for the input text.
+            this.Bind(ViewModel, vm =>
+                    vm.InputText, v => v.CreateFolderNameTextBox.Text)
                 .DisposeWith(disposable);
 
```

Comment:
I can't highlight `UpdateView` but that method should accept the `SelectableDirectoryNodeStatus` value from the `WhenAnyValue(view => view.ViewModel!.Status)` observable.